packages/bento-design-system/src/util/useCreatePortal.tsx

```diff
@@ -2,8 +2,13 @@ import { ReactPortal } from "react";
 import { createPortal as createReactPortal } from "react-dom";
 import { Children } from "./Children";
 import { useIsSSR } from "@react-aria/ssr";
+import { BentoThemeProvider, useBentoTheme } from "../BentoThemeContext";
 
 export function useCreatePortal(): (children: Children) => ReactPortal | null {
   const isSSR = useIsSSR();
-  return (children) => (isSSR ? null : createReactPortal(children, document.body));
+  const theme = useBentoTheme();
+  return (children) => {
+    const content = <BentoThemeProvider value={theme}>{children}</BentoThemeProvider>;
```

Comment:
we need to wrap the Portal children inside a new `BentoThemeProvider`, otherwise the Portal content won't inherit the style from the Box we add with `BentoThemeProvider`